database/migrations/2022_07_09_130303_create_webauthn_credentials.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+/**
+ * @see \Laragear\WebAuthn\Models\WebAuthnCredential
+ */
+class CreateWebauthnCredentials extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up(): void
+	{
+		Schema::dropIfExists('web_authn_credentials');
+		Schema::create('webauthn_credentials', static function (Blueprint $table): void {
+			static::defaultBlueprint($table);
```

Comment:
Just currious: Why this additional indirection? All our other migrations simply have the table definition in place. Why this change of heart?

Moreover, the function name `defaultBlueprint` is not telling anything actually.

Reply:
It is the result of :
`php artisan vendor:publish --provider="Laragear\WebAuthn\WebAuthnServiceProvider" --tag="migrations"`

Reply:
OK. We can leave it like that.